pkgs/tools/inputmethods/fcitx5/fcitx5-bamboo.nix

```diff
@@ -0,0 +1,42 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, extra-cmake-modules
+, fcitx5
+, gettext
+, go
+}:
+stdenv.mkDerivation rec {
+  pname = "fcitx5-bamboo";
+  version = "1.0.4";
+
+  src = fetchFromGitHub {
+    owner = "fcitx";
+    repo = "fcitx5-bamboo";
+    rev = version;
+    hash = "sha256-EcpuZN2JU6HSuiQgBPBsoYftdHypiyFlrUxDBlVW6eo=";
+    fetchSubmodules = true;
+  };
+
+  nativeBuildInputs = [
+    cmake
+    gettext
+    go
+  ];
```

Comment:
```suggestion
    extra-cmake-modules
  ];
```

Reply:
why

Reply:
yeah this is just completely wrong, the build breaks with this change